pkgs/development/python-modules/wxPython/3.0.nix

```diff
@@ -43,6 +45,15 @@ buildPythonPackage rec {
     # this check is supposed to only return false on older systems running non-framework python
     substituteInPlace src/osx_cocoa/_core_wrap.cpp \
       --replace "return wxPyTestDisplayAvailable();" "return true;"
+  '' + lib.optionalString (!stdenv.isDarwin) ''
+    substituteInPlace wx/lib/wxcairo.py \
+      --replace 'cairoLib = None' 'cairoLib = ctypes.CDLL("${cairo}/lib/libcairo.so")'
```

Comment:
Darwin (typically) needs `.dylib` instead of `.so`. It would be nice if there was a `stdenv.libextension` attribute that would return the correct extension for the current arch.